plugins/source/aws/codegen/recipes/ses.go

```diff
@@ -10,97 +11,122 @@ import (
 
 func SESResources() []*Resource {
 	resources := []*Resource{
-
 		{
-			SubService:          "templates",
-			Struct:              &models.Template{},
-			SkipFields:          []string{},
-			PreResourceResolver: "getTemplate",
+			SubService:  "active_receipt_rule_sets",
+			Struct:      &ses.DescribeActiveReceiptRuleSetOutput{},
+			Description: `https://docs.aws.amazon.com/ses/latest/APIReference/API_DescribeActiveReceiptRuleSet.html`,
+			SkipFields:  []string{"Metadata", "ResultMetadata"},
 			ExtraColumns: append(
-				defaultRegionalColumns,
-				[]codegen.ColumnDefinition{
-					{
-						Name:     "arn",
-						Type:     schema.TypeString,
-						Resolver: `resolveSesTemplateArn`,
-						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
-					},
-				}...),
+				defaultRegionalColumnsPK,
+				codegen.ColumnDefinition{
+					Name:     "name",
+					Type:     schema.TypeString,
+					Resolver: `schema.PathResolver("Metadata.Name")`,
+					Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				},
+				codegen.ColumnDefinition{
+					Name:     "created_timestamp",
+					Type:     schema.TypeTimestamp,
+					Resolver: `schema.PathResolver("Metadata.CreatedTimestamp")`,
+				},
+			),
 		},
 		{
 			SubService:          "configuration_sets",
 			Struct:              &sesv2.GetConfigurationSetOutput{},
 			Description:         "https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_GetConfigurationSet.html",
-			SkipFields:          []string{"ConfigurationSetName", "ResultMetadata"},
+			SkipFields:          []string{"ResultMetadata"},
```

Comment:
Can we split the refactoring changes (using primary keys instead of skip fields, using name transformer instead of skip fields, etc) from the code that adds the new resources?

Coupling multiple change in the same PR makes it hard to review the resources that were added